src/metabase/types.clj

```diff
@@ -158,6 +158,10 @@
 (derive :type/Country :type/Address)
 (derive :type/ZipCode :type/Address)
 
+;;; Structured
+
+(derive :type/Structured :type/*)
```

Comment:
Relevant existing types such as `:type/SerializedJSON` should derive from this.

Reply:
Serialized JSON is a string? Perhaps it should remain as text and a new `:type/JSON` should exist for json types. Presumably these are actually structured data like xml nodes and json types rather than textual representations of these structures?